DuckDuckGo/Tab/Model/Tab.swift

```diff
@@ -1328,9 +1328,12 @@ extension Tab {
 
 extension Tab: OnboardingNavigationDelegate {
     func searchFor(_ query: String) {
-        guard let url = URL.makeSearchUrl(from: query) else { return }
-        let request = URLRequest(url: url)
-        self.webView.load(request)
+        if let url = URL(string: query), url.isDuckDuckGoSearch {
```

Comment:
Probably making a comment here would help in the future to remember why we do this